web/app/scripts/src/directives/form.es.js

```diff
@@ -39,11 +50,40 @@ module.directive('surveysForm', [
                             // been set.
                             if (Object.keys(scope.model).length === 0) {
                                 scope.model = FieldsetParser.parseFormToModel(structure);
+                                countQuestionsTotal();
                             }
                         });
                     }
                 });
 
+                // Check and count the number of questions answered.
+                const checkAnswered = function (answers) {
```

Comment:
Comments usually try to make up for code that is not clear. In this case the comment can be easily removed by giving a better name to the function.